handler_test.go

```diff
@@ -43,7 +58,8 @@ func TestStateChangeHandler(t *testing.T) {
 	// Use ChanHandler
 	scCh := make(chan StateChange)
 	errCh := make(chan error)
-	chHand := NewChanHandler(scCh, errCh)
+	lpHandler := &logPositionHandler{}
```

Comment:
If you don't use `lpHandler` elsewhere, maybe you could pass `&logPositionHandler{}` directly in the `NewChanHandler()` call. True for 2 other occurrences in this test file.